src/chrome/content/rules/Racked.com.xml

```diff
@@ -1,16 +1,46 @@
 <!--
 	For other Vox Media coverage, see Vox.com.xml.
 
-	Problematic subdomains:
+	Problematic hosts in *racked.com:
 
-		- www *
+		- www.chicago (no https)
+		- comments.chicago (no https)
+		- comments.boston (no https)
+		- m.la (no https)
+		- comments.dc (no https)
+		- m.chicago (no https)
+		- m (no https)
+		- comments.la (no https)
+		- comments.miami (no https)
+		- comments (no https)
+		- www.ny (no https)
+		- m.ny (no https)
+		- comments.philly (no https)
+		- comments.ny (no https)
+		- www.philly (no https)
+		- comments.sf (no https)
+		- m.sf (no https)
+		- comments.vegas (no https)
```

Comment:
Please group the list by errors instead of writing `(no https)` to each line. Please be also more specific what "no https" means.

Reply:
To me `(no https)` means port 443 is refused at TCP level, but port 80 works, though I can see how that would be ambiguous.